lib/config/configuration.go

```diff
@@ -1208,17 +1209,33 @@ func applyWindowsDesktopConfig(fc *FileConfig, cfg *service.Config) error {
 	}
 	ldapPassword, err := os.ReadFile(fc.WindowsDesktop.LDAP.PasswordFile)
 	if err != nil {
-		return trace.WrapWithMessage(err, "loading LDAP password from file %v",
+		return trace.WrapWithMessage(err, "error loading LDAP password from file %v",
 			fc.WindowsDesktop.LDAP.PasswordFile)
 	}
+
+	var cert *x509.Certificate
+	if !fc.WindowsDesktop.LDAP.SkipVerifyCA {
```

Comment:
I think we should log a warning if `CAFile` is set and `SkipVerifyCA` is true.

While you may want to do that during testing or if your CA is not valid, there's a good chance that's an error and a nice warning might help users discover what's wrong.

Reply:
Good point, though because it's vital to security I think we should throw an error. During testing you can just comment that line out of the config.

Reply:
Ok, that's fine with me.